ccm_web/package.json

```diff
@@ -26,9 +26,13 @@
     "@nestjs/config": "^0.6.3",
     "@nestjs/core": "^7.6.15",
     "@nestjs/platform-express": "^7.6.15",
-    "@nestjs/swagger": "^4.8.0",
     "@nestjs/sequelize": "^0.2.0",
+    "@nestjs/swagger": "^4.8.0",
+    "class-transformer": "^0.4.0",
+    "class-validator": "^0.13.1",
```

Comment:
I've noticed some `npm install` errors about these libraries being peer dependencies.
```
npm WARN @nestjs/mapped-types@0.4.0 requires a peer of class-transformer@^0.2.0 || ^0.3.0 || ^0.4.0 but none is installed. You must install peer dependencies yourself.
npm WARN @nestjs/mapped-types@0.4.0 requires a peer of class-validator@^0.11.1 || ^0.12.0 || ^0.13.0 but none is installed. You must install peer dependencies yourself.
```
I decided to add them here. It's not strictly related to this issue, but something that was bothering me.
My guess is one of the Nest libraries we are using wants them because of this: https://docs.nestjs.com/pipes#class-validator

Reply:
agreed